pytorch_pfn_extras/training/trigger_util.py

```diff
@@ -10,7 +13,21 @@ def __call__(self, manager):
         raise NotImplementedError
 
 
-def get_trigger(trigger):
+class _CallableTrigger(Trigger):
```

Comment:
Could you split this to separate PR?

Reply:
Split in https://github.com/pfnet/pytorch-pfn-extras/pull/205.